source/browseMode.py

```diff
@@ -1281,7 +1288,6 @@ def _set_selection(self, info, reason=controlTypes.REASON_CARET):
 		if reason == controlTypes.REASON_FOCUS:
 			self._lastCaretMoveWasFocus = True
 			focusObj = api.getFocusObject()
-			self._lastFocusableObj = None

```

Comment:
Should `self._objPendingFocusBeforeActivate` also be set to None here?

Reply:
I'm not entirely sure why `_lastFocusableObj` was set to None there. I suspect that might have been what was breaking the tab key. Of course, that won't happen with the new implementation, but I also don't know that there's any need to clear it here either.